locale/en.yml

```
@@ -66,6 +66,7 @@ flarum-tags:
 
     # These translations are used by the Choose Tags modal dialog.
     choose_tags:
+      bypass_requirements: Bypass Tag Requirements
```

This casing is more consistent

```suggestion
      bypass_requirements: Bypass tag requirements
```